src/neuroconv/datainterfaces/ecephys/spikeglx/spikeglxdatainterface.py

```diff
@@ -3,9 +3,10 @@
 import json
 
 from pynwb.ecephys import ElectricalSeries
+from pydantic import FilePath
```

Comment:
Opinion: should each interface call something like

i) `from pydantic import FilePath`

or

ii) `from ....utils import FIlePath` (and the `pydantic` import being done in `neuroconv.utils.json_schema` and exposed to `neuroconv.utils`

?

Reply:
I'd rather i. It's simpler and clearer to the user that we are relying on pydantic